client/src/components/Common/Auth/useWithAccount.test.js

```diff
@@ -58,9 +58,7 @@ describe('useWithAccount', () => {
 	it('Should return nothing', () => {
 		getConnectedAccountChromeLocalStorage.mockResolvedValue({
 			publicKey: 'abc',
-			loginOptions: {
-				userHashingOptions: '1',
-			},
+			loginOptions: {},
```

Comment:
be careful when accepting this `{ }` value as empty object. It still a truthy value in Js.
Just friendly reminder that this will pass conditional like:

<img width="429" alt="image" src="https://user-images.githubusercontent.com/610234/198320106-3f63b73d-aad6-4780-8fc7-67eab21ec087.png">

or this code never runs:
`!({}) && console.log(">>x")`